monai/networks/blocks/aspp.py

```diff
@@ -37,8 +37,8 @@ def __init__(
         conv_out_channels: int,
         kernel_sizes: Sequence[int] = (1, 3, 3, 3),
         dilations: Sequence[int] = (1, 2, 4, 6),
-        norm_type=Norm.BATCH,
-        acti_type=Act.LEAKYRELU,
+        norm_type: Optional[Union[Tuple, str]] = "BATCH",
+        acti_type: Optional[Union[Tuple, str]] = "LEAKYRELU",
```

Comment:
What's wrong with leaving the right sight of the equals as they were? E.g., `Norm.BATCH` and `Act.LEAKYRELU`?

Reply:
Nothing wrong, I just try to use the same format as in https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/blocks/convolutions.py#L104

Reply:
Ok, thanks!